runtime/cere-dev/src/lib.rs

```diff
@@ -126,7 +128,7 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
 	// and set impl_version to 0. If only runtime
 	// implementation changes and behavior does not, then leave spec_version as
 	// is and increment impl_version.
-	spec_version: 43000,
+	spec_version: 42002,
```

Comment:
`spec_version` in `dev` is `43000`. Here should be `43001`. Please, update

Reply:
Updated